javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/scripts.js

```diff
@@ -0,0 +1,231 @@
+
+
+function app() {
+
+//  Songs array creation
+
+const songs = [
+    { title: "Scratching The Surface", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",artist: "Quincy Larson", duration: "4:25", src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/scratching-the-surface.mp3"},
+    { title: "Can't Stay Down", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "4:15" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cant-stay-down.mp3"},
+    { title: "Still Learning", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:51" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/still-learning.mp3"},
+    { title: "Cruising for a Musing", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:34" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cruising-for-a-musing.mp3"},
+    { title: "Never Not Favored", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:35" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/never-not-favored.mp3"},
+    { title: "From the Ground Up", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:12" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/from-the-ground-up.mp3"},
+    { title: "Walking on Air", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:25" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/walking-on-air.mp3"},
+    { title: "Can't Stop Me. Can't Even Slow Me Down.", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:52" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cant-stop-me-cant-even-slow-me-down.mp3"},
+    { title: "The Surest Way Out is Through", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:10" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/the-surest-way-out-is-through.mp3"},
+    { title: "Chasing That Feeling", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "2:43" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/chasing-that-feeling.mp3"},
+  ];
```

Comment:
This might be a personal preference but I think it would be easier to read with this prettier formatting.

```suggestion
const songs = [
  {
    title: "Scratching The Surface",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "4:25",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/scratching-the-surface.mp3",
  },
  {
    title: "Can't Stay Down",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "4:15",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cant-stay-down.mp3",
  },
  {
    title: "Still Learning",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "3:51",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/still-learning.mp3",
  },
  {
    title: "Cruising for a Musing",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "3:34",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cruising-for-a-musing.mp3",
  },
  {
    title: "Never Not Favored",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "3:35",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/never-not-favored.mp3",
  },
  {
    title: "From the Ground Up",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "3:12",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/from-the-ground-up.mp3",
  },
  {
    title: "Walking on Air",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "3:25",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/walking-on-air.mp3",
  },
  {
    title: "Can't Stop Me. Can't Even Slow Me Down.",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "3:52",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cant-stop-me-cant-even-slow-me-down.mp3",
  },
  {
    title: "The Surest Way Out is Through",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "3:10",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/the-surest-way-out-is-through.mp3",
  },
  {
    title: "Chasing That Feeling",
    cover:
      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
    artist: "Quincy Larson",
    duration: "2:43",
    src: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/chasing-that-feeling.mp3",
  },
];

```